### DISCUSSION TITLE
New installation not installing on Raspberry PI 3B Plus ???

### DISCUSSION THREAD

**Discussion by Infernoken76** [SBFspot/SBFspot#651] [Q&A] [answered]
Hi,

I tried to install a new instance of SBFspot after my SD-Card corrupted.

Installed on a new SD Card the Raspberry PI OS Lite (32-bit) - Debian Bullseye without desktop

OS installer correctly and I'm able to access through SSH...

But when trying the install via sbfspot-config using the command 
`curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config|sudo bash
`
It simply gets stuck...

See the picture...
What am I doing wrong ???

![image](https://github.com/SBFspot/SBFspot/assets/128485046/b6e9fd66-6c82-4839-8e83-ebdbbd041d8d)

**Comment by sbf-** [comment]
> What am I doing wrong ???

Nothing :-)

Try this:

```
lsb_release -si
```
and
```
wget --spider "https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/tzlatlon"
```
and
```
dpkg -s "whiptail"
```

**Reply by Infernoken76** [reply]
Thanks for the suggestion, but did not work...

I get the following:
![image](https://github.com/SBFspot/SBFspot/assets/128485046/1d675653-dd16-421d-a758-885e40c5deaf)

So get's stuck after
wget --spider "https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/tzlatlon"

Tried to continue with whiptail and the again SBFSpot config, but no go...

![image](https://github.com/SBFspot/SBFspot/assets/128485046/7559bb70-45d9-4dfe-ac7c-4d79dcca295c)

Tried with again a new install, new SD-Card and another Raspberry PI 3B Plus...
Exact same issue...  Something not compatible with the latest raspbian release???

**Reply by Infernoken76** [reply, marked as answer]
Seem to have found a solution?   Saw that the above also failed when using IPv6, so change the Raspberry PI config to IPv4 only:

EDIT the  etc/sysctl.conf with the following:

net.ipv6.conf.all.disable_ipv6 = 1
net.ipv6.conf.default.disable_ipv6 = 1
net.ipv6.conf.lo.disable_ipv6 = 1

And now the sbfspot-config continues ???  Don't know what happened, but it installs.
